tools/cli.js

```diff
@@ -286,7 +342,15 @@ lineProcessor.connect = (tokens, callback) => {
   if (appName === undefined) {
     return callback(reportMissingArgument('Application name'));
   }
-  commandProcessor.connect(scheme, host, port, appName, (err) => {
+  let interfaces = [];
+  if (args[2]) {
+    try {
+      interfaces = jstp.parse('[' + args[2] + ']');
```

Comment:
I find it a bit odd to be forced to put quotes around interface names in the `connect` command while I don't have to do that for an application name.